packages/test-viewer/vite.config.ts

```diff
@@ -15,7 +15,7 @@ export default defineConfig({
                 {
                     src: path.join(
                         require.resolve(
-                            "@doenet/doenetml-worker/CoreWorker.js",
+                            "@doenet/doenetml-worker/CoreWorker.iife.js",
```

Comment:
It would be really nice if it didn't have the `iife` suffix. Then these files wouldn't need changing...There should be a viteConfig option for that (you might have to override the naming operation...)

Reply:
I am generally in the camp of minimizing the size of diffs, but in this case I think this default behavior makes sense to me, we have other artifacts we produce out of one of our packages that are using modules and they end up in `.js` files. So while I'm not going to argue hard one way or the other, I think for this weird case having the difference  highlighted more explicitly seems useful.